plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -2677,4 +2705,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
Final newline missing
```suggestion
    main()
```

Reply:
I removed that because there was an error in pylint:
```
ERROR: Found 1 pylint issue(s) which need to be resolved:
ERROR: plugins/modules/azure_rm_virtualmachine.py:2709:0: trailing-newlines: Trailing newlines
```